inventory/host_vars/www.openfoodnetwork.org.uk/config.yml

```diff
@@ -19,3 +19,10 @@ custom_hba_entries:
 
 enable_nginx_logging: true
 enable_rails_apm: "true" # has to be explicitly defined as a string due to some datadog bug
+
+# Images settings
+attachment_url: "/spree/products/:id/:style/:basename.:extension"
+attachment_styles: "{\"mini\":[\"48x48#\",\"\"],\"small\":[\"100x100#\",\"\"],\"product\":[\"240x240\u003E\",\"\"],\"large\":[\"600x600\u003E\",\"\"]}"
+attachment_path: "public/spree/products/:id/:style/:basename.:extension"
+attachment_default_style: "product"
+attachment_default_url: "/spree/products/:id/:style/:basename.:extension"
```

Comment:
it's basically Katuma the one that doesn't use S3, right? In that case, I'd like to avoid that much duplication (we have quite a few already), which will ease adding new servers as well. We can just treat Katuma as the single exception and probably move it to S3 soon too. It's worth the simplification it will bring IMO.

tl;dr: I would make these settings the default value in roles/app/templates/application.yml.j2 (I believe all servers except Katuma share the same values) and use the var only if defined (just Katuma will).

Reply:
I think we should stop using S3 and I dont think S3 should be the default. Relates to https://github.com/openfoodfoundation/openfoodnetwork/issues/5078

Anyway, it's a valid point, I'll fetch all the values from all the servers and see what's repeated. If all the servers have the same values, they should go to roles/app/templates/application.yml.j2 :+1:  I'd do this only to non-S3 specific settings though. I'll let you know when I have all the configs and then we decide, ok?

Reply:
yes, treating Katuma's as an exception would indeed make things more manageable and easier to migrate from S3 to a better alternative.

Reply:
ok, this is done in the new commits :+1: please have a look.